golang/testsuite/testsuite_impl/network_partition_test/network_partition_test_.go

```diff
@@ -90,20 +88,26 @@ func (test NetworkPartitionTest) Run(network networks.Network) error {
 	// Go doesn't have generics so we have to do this cast first
 	castedNetwork := network.(*networks.NetworkContext)
 
-
 	logrus.Info("Partitioning API and datastore services off from each other...")
 	if err := repartitionNetwork(castedNetwork, true, false); err != nil {
 		return stacktrace.Propagate(err, "An error occurred repartitioning the network to block access between API <-> datastore")
 	}
 	logrus.Info("Repartition complete")
 
+	datastoreConfigFactory := datastore.NewDatastoreContainerConfigFactory(test.datstoreImage)
```

Comment:
Why is this guy here?

Reply:
Because configFactory is the second argument of `GetSeviceContext`but we will be able to remove it when I add `volumeMountPoint`inside K Core. This will be my next task.

Reply:
awesome yes